pandas/core/generic.py

```diff
@@ -4108,8 +4108,11 @@ def _consolidate(self, inplace=False):
             return self._constructor(cons_data).__finalize__(self)
 
     def consolidate(self, inplace=False):
-        """
-        DEPRECATED: consolidate will be an internal implementation only.
+        """Compute NDFrame with "consolidated" internals (data of each dtype
```

Comment:
For future reference, please never use 'NDFrame' is user-facing docstrings. Users should not be aware what that means.